doc/dev/build_archived_sensfuncs.rst

```diff
@@ -3,26 +3,60 @@
 
 .. _build_archived_sensfuncs:
 
-How to build archived sensitivity functions
-===========================================
+
+Creating archival sensitivity functions
+=======================================
+
+Archival sensitivity functions for instruments can be created by combining 
+multiple sensivity functions from different exposures at different 
+grating tilts to get a wider wavelength coverage. The steps in this
+process are typically:
+
+#. Choose sensitivity functions for grating. The functions should have good wavelength coverage,
+   and low airmass.
+
+#. Start with the bluest sensitivity function and translate the next function up/down to roughly match.
+   One or both function may be truncated to fit.
+
+#. If the gap between sensitivity function isn't smooth (for example on a detector boundary) 
+   a polynomial fit can be used to join them.
+
+There are helper functions in $PYPEIT_DEV/sensfunc_archive/stitcutils.py intended to make this easier. See
+$PYPEIT_DEV/sensfunc_archive/create_deimos_sensfuncs.py $PYPEIT_DEV/sensfunc_archive/stitchdeimos.py for examples of how this was done
+for DEIMOS.
+
+The DEIMOS archived sensitivity functions
+-----------------------------------------
+
+*Disclaimer*:
+The DEIMOS archival sensitivity functions do not provide an absolute flux 
+calibration.  Instead, they are only intended to remove the instrumental 
+response, providing a relative flux calibration up to some unknown 
+normalization.
+
+
+Creating all of the DEIMOS sensitivity functions
```

Comment:
"Creating DEIMOS stitched sensitivity functions for every grating"

Reply:
done